index.html

```diff
@@ -8,62 +8,194 @@
   <link rel="stylesheet" href="index.css">
   <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
 </head>
-<body>   
-  <header>
-	<h1 class="header">John Doe</h1>
-	  <nav>
-		<img src="assets/images/menuIcon.png" alt="" class="icon menu-icon">
-		  <div class="nav-links" style="display: none;">
-			<a href="" id="portfolio">Portfolio</a>
-			<a href="" id="about">About</a>
-			<a href="" id="contact">Contact</a>
+<body>
+    <header>
+            <h1 class="header">John Doe</h1>
+    
+            <nav>
+                <img src="assets/images/menuIcon.png" alt="" class="icon menu-icon">
+                <div class="nav-links" style="display: none;">
+                    <a href="" id="portfolio">Portfolio</a>
+                    <a href="" id="about">About</a>
+                    <a href="" id="contact">Contact</a>
+                </div>
+            </nav>
+    </header>
+    <main>
+        <section class="hero">
+            <span class="header">Hey There.
```

Comment:
[OPTIONAL] I suggest you change this to a `<h1>` element. This is is the title of your page and it should use a header tag

Reply:
okay sir.

Reply:
done @Nyame-Wolf